pyfftw/pyfftw.pyx

```diff
@@ -1394,6 +1405,10 @@ cdef class FFTW:
         copy the returned array.
         '''
 
+        if ortho and normalise_idft:
+            raise ValueError('Invalid options: ortho and normalise_idft cannot'
+                             ' both be True.')
+
```

Comment:
Yeah, I concur. I think I'll accept as is for the time being and review a bit later. We need an API consolidation at some point - there are a few things I've been mulling over that need changing.